### ISSUE TITLE
Missing logs incase of incorrect model name

### ISSUE THREAD

**Issue by dugarsumit** [triton-inference-server/server#3831]
**Description**
I have created a python model which can call different onnx models after some preprocessing based on some conditions. 

1. When I pass an incorrect python `model_name` to the triton client there are no error logs in the triton server.
2. But if my python model calls an incorrect onnx model then I can see errors in the triton server.

Triton Information
TRITON_VERSION=2.14.0

Are you using the Triton container or did you build it yourself?
TRITON_CONTAINER_VERSION=21.09

**Expected behavior**
I expect to see a consistent behaviour where error logs get printed in both the above-mentioned cases.

**Comment by Tabrizian** [comment]
The error is not logged but you can check whether your BLS request had error using the `has_error` method of the inference response.  In case it had an error, the error message is included in the `infer_response.error().message()`.

**Comment by dugarsumit** [comment]
yes, I can get the error as you mentioned. But I was curious if there's any specific reason for not logging these errors. The reason I am asking it is from the point of view of monitoring and alerting. I want to trigger an alert system based on the errors that I get in the triton server.

**Comment by Tabrizian** [comment]
I see. I have filed an enhancement ticket for this.
